### PR TITLE
Expose cleaned prow job metrics to Prometheus

### PR DESCRIPTION
**Pull request by hongkailiu**
Signed-off-by: Hongkai Liu <hongkliu@redhat.com>

### CODE REVIEW

**Comment by petr-muller** [comment]
LGTM

File: prow/cmd/sinker/main.go [hunk 1]
```diff
@@ -154,20 +157,24 @@ type controller struct {
...
 }
 
 // Prometheus Metrics
 var (
 	sinkerMetrics = struct {
-		podsCreated      prometheus.Gauge
-		timeUsed         prometheus.Gauge
-		podsRemoved      *prometheus.GaugeVec
-		podRemovalErrors *prometheus.GaugeVec
+		podsCreated            prometheus.Gauge
```
Comment (stevekuznetsov) [inline prow/cmd/sinker/main.go]: Could we also keep track of how many ProwJobs exist?

File: prow/cmd/sinker/main.go [hunk 2]
```diff
@@ -53,8 +53,11 @@ const (
 	// TODO(fejta): require setting this explicitly
 	defaultConfigPath = "/etc/config/config.yaml"
 
-	reasonAged     = "aged"
-	reasonOrphaned = "orphaned"
+	reasonPodAged     = "aged"
+	reasonPodOrphaned = "orphaned"
+
+	reasonProwJobOldOnce     = "old-once"
```
Comment (stevekuznetsov) [inline prow/cmd/sinker/main.go]: I think we could shorten this to just `"old"`

Comment (stevekuznetsov) [reply prow/cmd/sinker/main.go]: Or, use `"aged"` to mirror what we do for Pods

Comment (hongkailiu) [reply prow/cmd/sinker/main.go]: I will do the revision tonight according to your comments.
Please do NOT approve until then.
Thanks.

Comment (hongkailiu) [reply prow/cmd/sinker/main.go]: OhOh ... that was too late. :)

Comment (stevekuznetsov) [reply prow/cmd/sinker/main.go]: Sorry, didn't realize we had changed GitHub approval to do LGTM and approve already
